security-privacy-questionnaire.md

```diff
@@ -1,99 +0,0 @@
-# Security and Privacy Questionnaire for the Push API
```

Comment:
Not sure whether this is a necessary part of this.  If it's dead (it certainly looks that way), I would have expected this to be in a separate changeset.  So I just want to make sure that this is deliberate.

Reply:
Yeah, I was being lazy. I'll resend. It's not dead - just moved to the wiki https://github.com/w3c/push-api/wiki/Security-and-Privacy-Questionnaire-for-the-Push-API